pkgs/misc/tpm2-pkcs11/default.nix

```diff
@@ -0,0 +1,79 @@
+{ stdenv, lib, fetchFromGitHub, writeText
+, pkgconfig, autoreconfHook, autoconf-archive, makeWrapper, patchelf
+, tpm2-tss, tpm2-tools, opensc, openssl, sqlite, python37, glibc, libyaml
+, abrmdSupport ? true, tpm2-abrmd ? null
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tpm2-pkcs11";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "tpm2-software";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256:06kpf730al50xv1q53ahycky3im23ysrqp40libls4k24zxs9ha2";
+  };
+
+  patches = lib.singleton
+    (writeText "${pname}-${version}-configure-ac-version.patch"
+      (builtins.replaceStrings ["VERSION"] [version]
+        (builtins.readFile ./0001-configure-ac-version.patch)));
```

Comment:
You can use `pkgs.substituteAll { src = ./foo.patch; VERSION = version; }` instead, using `@VERSION@` in the patch then.